docs/installation.rst

```diff
@@ -4,84 +4,92 @@
 Installation
 ************
 
-Before installing TARDIS, please check its :ref:`requirements
-<requirements_label>`. We provide :ref:`instructions <anaconda_inst_label>` for installing TARDIS using 
-Anaconda. Once you have installed TARDIS, check out the "Input/Output" section (in sidebar) for instructions
-regarding how to perform simple TARDIS calculations.
 
-.. _requirements_label:
+.. warning::
+    
+    - TARDIS is only supported on macOS and GNU/Linux. Windows users can run TARDIS 
+      from our official Docker image (*coming soon*), `WSL <https://docs.microsoft.com/en-us/windows/wsl/>`_ 
+      or a Virtual Machine.
 
+    - TARDIS packages and dependencies are distributed only through the `conda <https://docs.conda.io/en/latest/>`_ 
+      package management system, therefore installation requires `Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ 
+      or `Miniconda <https://conda.io/projects/conda/en/latest/user-guide/install/index.html>`_
+      to be installed on your system.
 
-.. note::
 
-    - TARDIS is only compatible with Python >=3.6
-    - TARDIS only supports MacOS and Linux.
-    - We strongly recommend installing TARDIS within an Anaconda environment and to always use the latest GitHub development version.
+Development version
+===================
 
+We encourage users to use the latest TARDIS development version by reproducing the following steps:
 
-Requirements
-============
+1. Download the latest dependency file for your platform (`linux` or `osx`) from our 
+   `releases page <https://github.com/tardis-sn/tardis/releases>`_.
 
-You can see a list of all the requirements of TARDIS in the `environment definition file <https://raw.githubusercontent.com/tardis-sn/tardis/master/tardis_env3.yml>`_.
+  ::
 
-TARDIS is using Astropy's excellent installation helpers and thus uses similar
-instructions to Astropy.
+    $ wget -q https://github.com/tardis-sn/tardis/releases/latest/download/conda-{platform}-64.lock
 
-.. _anaconda_inst_label:
+2. Create and activate the `tardis` environment.
 
-Installing TARDIS with Anaconda
-===============================
+  ::
 
-We highly recommend using the `Anaconda <https://www.anaconda.com/>`_ Python environment to install TARDIS (or
-any other scientific packages for that matter). Anaconda has the advantage of
-being an isolated environment that can be set to be the default one, but by no
-means will mess with your other environments. It will also work on computers
-where ``root``-rights are not available. Use these `instructions
-<http://docs.continuum.io/anaconda/install.html>`_ to install Anaconda on your
-machine. The next step is to create an environment for TARDIS that contains all
-of the necessary packages (this ensures that TARDIS requirements won't clash
-with any other Python installs on disc):
+    $ conda create -n tardis --file conda-{platform}-64.lock
```

Comment:
This isn't rendering properly

Reply:
This is not a templating issue. You should use `linux` or `osx` where `{platform}` according to your OS. Maybe it's not clear enough.

Reply:
Reformulated that phrase to avoid confusion between newbie users. Thanks.